js/ui/overlay/ui.overlay.js

```diff
@@ -240,6 +243,14 @@ const Overlay = Widget.inherit({
         });
     },
 
+    ctor: function(element, options) {
+        this.callBase(element, options);
+
+        if(options && options.copyRootClassesToWrapper && !options.hideCopyClassesDeprecationWarning) {
+            errors.log('W0001', this.NAME, 'copyRootClassesToWrapper', '21.2', 'Use the "wrapperAttr" option instead');
```

Comment:
why `W001`? We should add a new warning.

Reply:
afaik `W001` is warning for all deprecated options